libs/net/traffic_generator.py

```
@@ -166,18 +166,33 @@ class PodTcpClient(BaseTcpClient):
         server_port (int): The port on which the server listens for connections.
         bind_interface (str): The interface or IP address to bind the client to (optional).
             If not specified, the client will use the default interface.
+        container (str): Container name to execute commands in.
+        netns (str): Network namespace to run iperf3 in (optional).
+            'ip netns exec' replaces itself with iperf3 via execvp,
+            so pgrep/pkill match by the bare iperf3 cmdline.
     """
 
-    def __init__(self, pod: Pod, server_ip: str, server_port: int, bind_interface: str | None = None):
+    def __init__(
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)